CommunicationLayer/CommDeviceControl/UdpMessageForwarder.cpp

```diff
@@ -26,27 +26,39 @@
 #include <QByteArray>
 #include <QDebug>
 #include <QHostAddress>
+#include <QTextCodec>
+
 
 #include "InfrastructureLayer/Settings/I_Settings.h"
 #include "UdpMessageForwarder.h"
 
+namespace
+{
+    const QString EXCHANGE_NAME = QString("amq.direct");
```

Comment:
Still could be discussion on whether this is the best exchange type to use, or whether it should be implemented in settings with other rabbitmq variables